MPChartExample/src/main/java/com/xxmassdeveloper/mpchartexample/LineChartActivity1.java

```diff
@@ -202,24 +201,26 @@ private void setData(int count, float range) {
             set1 = new LineDataSet(values, "DataSet 1");
 
             set1.setDrawIcons(false);
-
+            set1.setMode(LineDataSet.Mode.GRADIENT_CUBIC_BEZIER);
+            set1.setGradientColor(getResources().getColor(android.R.color.holo_green_dark),
+                    getResources().getColor(android.R.color.holo_blue_dark));
             // draw dashed line
```

Comment:
**Performance Issue**: Disabling the dashed line with 'set1.enableDashedLine(10f, 5f, 0f);' and modifying the line width from '1f' to '5f' could impact the chart's readability and visual appeal. <br> **Fix**: Consider the visual implications of these changes on the chart's readability. If the dashed line is essential for distinguishing data sets or if a thinner line width is preferred for aesthetics, revert these changes. <br> **Code Suggestion**: 
 ```
 -            set1.enableDashedLine(10f, 5f, 0f);
 +            /* set1.enableDashedLine(10f, 5f, 0f);*/
 -            set1.setLineWidth(1f);
 +            set1.setLineWidth(5f);
 ```